### ISSUE TITLE
Missing semi-colon in the `extendsHelper`

### ISSUE THREAD

**Issue by spratmannc** [Microsoft/TypeScript#27312]
The missing semi-colon after the declaration of the `extendStatics` variable/function (es2015.ts, line 4082) results in linting errors (JSHint).  For projects that contain both TypeScript and JavaScript source files, this presents a problem in CI builds, as its not always feasible to bypass semi-colon terminator requirements when linting the project.

It looks like this is just an oversight, since semi-colons are used consistently elsewhere.   

This would not show up as a build error since the helper is specified as a string of text.

**Expected behavior:**
The `extendsHelper` helper should include a semi-colon after declaring the `extendsStatic` variable.

**Actual behavior:**
The semi-colon is missing.

**Comment by spratmannc** [comment]
#27313 Should address this.

**Comment by PaulIvanov** [comment]
does this issue need to be closed?

**Comment by spratmannc** [comment]
No.  I'm still waiting on pull request #27313 to be reviewed.  Do I need to do something special to get it looked at?
